app/Enums/Semester.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum Semester: string
```

Comment:
I would drop this class and just use numeric semester numbers in the database.

Reply:
They could be romanized directly in model.

Reply:
fixed